lib/floki.ex

```diff
@@ -102,13 +104,20 @@ defmodule Floki do
 
   ## Options
 
+    * `:attributes_as_maps` - Change the behaviour of the parser to return the attributes
+      as maps, instead of a list of `{"key", "value"}`. Remember that maps are no longer 
+      ordered since OTP 26. Default to `false`.
```

Comment:
Maps were never ordered (rather, they never guaranteed a particular ordering). :)

```suggestion
      as maps, instead of a list of `{"key", "value"}`. Default to `false`.
```